docs/modeling/validate-code-with-layer-diagrams.md

```diff
@@ -223,6 +223,8 @@ The following section describes the syntax that is used in these errors, explain
 | DV1001: **Invalid namespace name** | This issue is reported on a code element associated with a layer which "Allowed Namespace Names" property does not contain the namespace in which this code element is defined. This is a naming constraint violation. Note that the syntax of "Allowed Namespace Names" is to be a semi-colon list of namespaces in which code elements associated with are layer are permitted to be defined. |
 | DV1002: **Dependency on unreferenceable namespace** | This issue is reported on a code element associated with a layer and referencing another code element defined in a namespace which is defined in the "Unreferenceable Namespace" property of the layer. This is a naming constraint violation. Note that the "Unreferenceable Namespaces" property is defined as a Semi-colon separated list of namespaces that should not be referenced in code elements associated with this layer. |
 | DV1003: **Disallowed namespace name** | This issue is reported on a code element associated with a layer which "Disallowed Namespace Names" property contains the namespace in which this code element is defined. This is a naming constraint violation. Note that the "Disallowed namespace name" property is defined as a Semi-colon separated list of namespaces in which code elements associated with this Layer should not be defined. |
+| DV2001: **Layer Diagram Presence** | This issue is reported on a project that do not include a dependency diagram file but refere the dependency validation analyzers. If Dependency Validation is not been use, you can remove "Microsoft.DependencyValidation.Analyzers" directly from Solution Explorer or supress this warning. To add a dependency diagram see [Create dependency diagrams from your code](../modeling/create-layer-diagrams-from-your-code.md). |
```

Comment:
Suggest edit of text to: 

This issue is reported on a project that does not include a dependency diagram file, but refers to the dependency validation analyzers. If Dependency Validation has not been used, you can remove "Microsoft.DependencyValidation.Analyzers" directly from Solution Explorer or suppress this warning. To add a dependency diagram see [Create dependency diagrams from your code](../modeling/create-layer-diagrams-from-your-code.md)